test/parallel/test-http2-many-writes-and-destroy.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const http2 = require('http2');
+const assert = require('assert');
+
+{
+  const server = http2.createServer((req, res) => {
+    req.pipe(res);
+  });
+
+  setTimeout(() => {
+    assert(false, 'should exit before 10s');
+  }, 10000).unref();
+
+  server.listen(0, () => {
+    const url = `http://localhost:${server.address().port}`;
+    const client = http2.connect(url);
+    const req = client.request({ ':method': 'POST' });
+
+    for (let i = 0; i < 4000; i++) {
+      req.write(Buffer.alloc(6));
+    }
+
+    req.on('close', common.mustCall(() => {
+      console.log('(req onclose)');
+      server.close();
+      client.close();
+    }));
+
+    req.once('data', () => req.destroy());
```

Comment:
I think this could be `common.mustCallAtLeast()`'ed, to make sure it's run?

Reply:
Since it's a `once` handler, it could be `common.mustCall()`.

If the test will fail without the handler running, then I'm OK omitting `common.mustCall()`. And I'm OK including it. Either way...

Reply:
fixed. added a mustCall, since it's a once.